rangeseekbar/src/main/java/org/florescu/android/rangeseekbar/RangeSeekBar.java

```diff
@@ -707,14 +724,18 @@ protected void onRestoreInstanceState(Parcelable parcel) {
 
     /**
      * Draws the "normal" resp. "pressed" thumb image on specified x-coordinate.
+     * alwaysActive variable overwrites activateOnDefaultValues
      *
-     * @param screenCoord The x-coordinate in screen space where to draw the image.
-     * @param pressed     Is the thumb currently in "pressed" state?
-     * @param canvas      The canvas to draw upon.
+     * @param screenCoord               The x-coordinate in screen space where to draw the image.
+     * @param pressed                   Is the thumb currently in "pressed" state?
+     * @param canvas                    The canvas to draw upon.
+     * @param areSelectedValuesDefault  Should show active state button if selected values are selected
      */
     private void drawThumb(float screenCoord, boolean pressed, Canvas canvas, boolean areSelectedValuesDefault) {
         Bitmap buttonToDraw;
-        if (!activateOnDefaultValues && areSelectedValuesDefault) {
+        if (alwaysActive) {
```

Comment:
Why did you need this? As far as I can tell, if the `alwaysActive` flag is on, the buttons never show as `disabled`